README.md

```diff
@@ -124,3 +126,19 @@ EOF
 23 0 * * * /home/centos/gh-snap/run_as_cronjob snap
 46 4 * * * /home/centos/gh-snap/run_as_cronjob purge
 ```
+
+### Docker
+
+Build a container from the included `Dockerfile`.
+
+You may want to specify some or all of the following environment
+variables with -e when you do a `docker run`:
+
+`AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN`, 
+`AWS_DEFAULT_REGION`, `GITHUB_ORGS`, `GITHUB_TOKEN`, `S3_BACKUP_BUCKET`
```

Comment:
Completely subjective but I think this would read better as an actually MD list.